localization/french/artifacts/wc_artifacts_l_french.yml

```
@@ -0,0 +1,24 @@
+﻿l_french:
+
+ terenas_crown_name:0 "Couronne de Lordaeron"
+ terenas_crown_desc:0 "Une couronne à pointes utilisée pour le couronnement des hauts rois de Lordaeron."
```

```suggestion
 terenas_crown_desc:0 "Une couronne à pointes utilisée pour le couronnement des Hauts-Rois de Lordaeron."
```
C'est plus écrit comme ça dans la trad officielle